Source/WebCore/platform/graphics/filters/FEColorMatrix.cpp

```diff
@@ -124,7 +124,7 @@ OptionSet<FilterRenderingMode> FEColorMatrix::supportedFilterRenderingModes() co
     if (FEColorMatrixCoreImageApplier::supportsCoreImageRendering(*this))
         modes.add(FilterRenderingMode::Accelerated);
 #endif
-#if HAVE(CGSTYLE_COLORMATRIX_BLUR)
+#if 0 && HAVE(CGSTYLE_COLORMATRIX_BLUR)
```

Comment:
Can you add a comment in here (and in FEGaussianBlur.cpp) to explain why we're doing `0 &&`. Without any context it looks odd.